contents/founders/negotiate-software-better.md

```diff
@@ -0,0 +1,91 @@
+
+_Put it in writing_. If someone isn’t willing to put something in writing, they will definitely not do it. You are not friends. There is no social obligation to help you out here. 
+
+> The PostHog way: at PostHog, the [technical account executive](/handbook/growth/sales/overview#how-we-define-roles) who sells you the software is _also_ the person who manages your account in the long term. They’ll get to know you and your company really well, and are financially incentivized to make sure you _stay_ happy. We also tell you up front exactly what to expect from [onboarding](/handbook/growth/sales/customer-onboarding).
+
+# Avoid the nasties in the legal docs
+
+You’ve negotiated hard and agreed on a price you’re happy with - hooray! Wait, but it’s not quite over yet - here are a couple of nasty bits in contracts that Saas companies will include by default: 
+
+- ‘We will renew your contract each year by default unless you give us 90 days notice of your intention not to renew’ - take. this. out. Your salesperson will make a lot of noise about getting that secret approval we talked about before, but you do not want to rely on a random calendar reminder you set yourself 9 months from now. 
+  - You also want to give yourself an opportunity to renegotiate and improve your contract. 
+- You don’t have the right to terminate the contract, only they do. You want this to be mutual - you want to be able to terminate the contract if they breach the terms. In practice, performance is the number one thing you will probably want to terminate for, so make sure some kind of SLA is included.
+- Automatic price increase by X% at renewal - easiest way to avoid this is not having auto renewal in the first place. However, if they _really_ won’t budge (which can be the case in very large companies), make sure that they haven’t hidden something dumb like ‘inflation + 10%’. Reasonable here is sub-5%. 
+
+> The PostHog way: you know the [special enterprise contract](https://docs.google.com/document/d/155w70ZAHecVZcDqTq2_415dvaq2Bk-8QlEOozjq1hG8/edit#heading=h.y38xfjgcg4xm) you asked for so finance can sign it? Yeah, it’s just our [terms](/terms) copied and pasted into a Google Doc to look more corporate. Read it in advance, not right at the end of a process when you are tired. And no, we don’t do auto-renewals. Unless you ask for it.
+
+# But Charles, I am le tired and can't be bothered to negotiate
+
+Maybe you’re already happy with the price or don’t really care to spend a bunch of time negotiating - that’s fine!
+
+You can use my Two-Step Low Effort Negotiation Framework for Busy People instead:
+
+- Ask ‘is this the best price you can offer?’
+- Stay completely silent (this is the hard part). 
+
+Sometimes you’ll get a decent chunk knocked off or some goodies thrown in - and you didn’t even care!
+
+> The PostHog way: the answer is always ‘yes’ because it’s all in our Handbook already - but we are very generous with [swanky merch](/merch) if you want it. 
```

Comment:
```suggestion
> The PostHog way: The answer is always ‘yes’ because it’s all in our handbook already - but we are very generous with [swanky merch](/merch) if you're nice to us. 
```